lib/internal/backend/file/file.go

```diff
@@ -260,7 +276,7 @@ func getImageID(file *os.File, dockerURL *common.ParsedDockerURL, name string, d
 
 			if dockerURL == nil {
 				dockerURL = &common.ParsedDockerURL{
-					OriginalName: "",
+					OriginalName: appName + ":" + tag,
```

Comment:
If you save an image then you get a manifest.json with a null RepoTags, and docker2aci can't do anything with it currently (errors out, even if you use the `-image` flag).